packages/synapse-react-client/src/components/AccountLevelBadges/AccountLevelBadges.tsx

```diff
@@ -0,0 +1,48 @@
+import { Box } from '@mui/material'
+import React from 'react'
+import { useGetUserBundle } from '../../synapse-queries/user/useUserBundle'
+import { SynapseConstants } from '../../utils'
+import { AccountLevelBadge } from '../AccountLevelBadge/AccountLevelBadge'
+import { ErrorBanner } from '../error/ErrorBanner'
+
+export type AccountLevelBadgesProps = {
+  userId: string
+}
+
+const CERTIFICATION_AND_VERIFICATION_BUNDLE_MASK =
+  SynapseConstants.USER_BUNDLE_MASK_IS_CERTIFIED |
+  SynapseConstants.USER_BUNDLE_MASK_IS_VERIFIED
+
+export const AccountLevelBadges: React.FunctionComponent<
+  AccountLevelBadgesProps
+> = ({ userId }: AccountLevelBadgesProps) => {
+  const {
+    data: userBundle,
+    isLoading,
+    error,
+  } = useGetUserBundle(userId, CERTIFICATION_AND_VERIFICATION_BUNDLE_MASK)
+
+  const hasBadges = userBundle?.isCertified || userBundle?.isVerified
+
+  if (isLoading) {
+    return <></>
```

Comment:
Not sure if you have the container/label hidden by default in https://github.com/Sage-Bionetworks/SynapseWebClient/pull/5253

If you do, consider showing one or two Skeleton badges during loading.

You might be able to easily get a badge of the correct size if you do 
```
<Skeleton>
  <AccountLevelBadge badgeType="certified" />
</Skeleton>
```

Reply:
Looking at your other PR more closely, since it's visible by default, I think the fragment is fine--it's kind of a judgement call since a lot of users won't have anything shown after loading.

Reply:
Oh good point! Since most users won't have a badge shown after loading, I'll keep it as is for now. If most users end up with badges in the future, then we can add the Skeleton!